channeldb/channel.go

```diff
@@ -2279,12 +2358,20 @@ func putChanCommitment(chanBucket *bbolt.Bucket, c *ChannelCommitment,
 }
 
 func putChanCommitments(chanBucket *bbolt.Bucket, channel *OpenChannel) error {
-	err := putChanCommitment(chanBucket, &channel.LocalCommitment, true)
+	// If this is a restored channel, then we don't have any commitments to
+	// write.
+	isRestoredChan := channel.hasChanStatus(ChanStatusRestored)
```

Comment:
what do you think about doing this here, this is the simplest change that comes to mind:
```golang
if channel.hasChanStatus(ChanStatusRestored) {
        return nil
}

err := putChanCommitment(
         chanBucket, &channel.LocalCommitment, false, true,
)
...
```
and then other calls to `putChanCommitment` can be invoked with a computed `isRestoredChan` and return a concrete error? It doesn't seem worth it to propagate an `ignoreRestoreError` bool all the way up to `syncChannel`, but in all other instances it looks like we assume `isRestoredChan=false`, in which case we would want that error

Reply:
Removed the extra param from `putChanCommitment` all together in the latest fix-up commit.